addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/api/SamsungTvService.java

```diff
@@ -8,71 +8,70 @@
  */
 package org.openhab.binding.samsungtv.internal.service.api;
 
-import java.util.List;
-
 import org.eclipse.smarthome.core.types.Command;
 
+import java.util.List;
+
 /**
  * Interface for Samsung TV services.
  *
  * @author Pauli Anttila - Initial contribution
  */
 public interface SamsungTvService {
 
-    /**
-     * Procedure to get service name.
-     * 
-     * @return Service name
-     */
-    public String getServiceName();
-
     /**
      * Procedure to get list of supported channel names.
-     * 
+     *
      * @return List of supported
      */
-    public List<String> getSupportedChannelNames();
+    List<String> getSupportedChannelNames();
 
     /**
      * Procedure for sending command.
-     * 
-     * @param listener
-     *            Event listener instance to handle events.
+     *
+     * @param channel the channel to which the command applies
+     * @param command the command to be handled
      */
-    public void handleCommand(String channel, Command command);
+    void handleCommand(String channel, Command command);
 
     /**
      * Procedure for register event listener.
-     * 
+     *
      * @param listener
      *            Event listener instance to handle events.
      */
-    public void addEventListener(ValueReceiver listener);
+    void addEventListener(EventListener listener);
 
     /**
      * Procedure for remove event listener.
-     * 
+     *
      * @param listener
      *            Event listener instance to remove.
      */
-    public void removeEventListener(ValueReceiver listener);
+    void removeEventListener(EventListener listener);
 
     /**
      * Procedure for starting service.
-     * 
+     *
      */
-    public void start();
+    void start();
 
     /**
      * Procedure for stopping service.
-     * 
+     *
      */
-    public void stop();
+    void stop();
 
     /**
      * Procedure for clearing internal caches.
-     * 
+     *
      */
-    public void clearCache();
+    void clearCache();
 
+    /**
+     * Is this a manual configured service
```

Comment:
Is "manual(ly) configured service really the right term here?
Once a Thing is in the system, there actually should not be any difference between a discovered one and a manually added one. From what I understand from your code, your use this flag as a fallback, when you cannot discover the device through UPnP, so you connect differently to it. But this can also happen with an automatically discovered device, correct?